projects/wc/src/app/components/development-settings/development-settings.component.scss

```
@@ -42,7 +48,8 @@
   font-weight: 400;
   margin-bottom: -0.125rem;
   color: var(--sapContent_LabelColor, #4a4a4a);
-  text-wrap: nowrap;
+  overflow-wrap: break-word;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)